.ci/post_review.py

```diff
@@ -0,0 +1,214 @@
+#! usr/bin/env python3
+
+# Automatically create review from clang-tidy report
+# Copyright (c) Ashar Khan <ashar786khan.at.gmail.com>
+
+
+# Boost Software License 1.0
+# Permission is hereby granted, free of charge, to any person or
+# organization obtaining a copy of the software and accompanying
+# documentation covered by this license (the "Software") to use,
+# reproduce, display, distribute, execute, and transmit the Software,
+# and to prepare derivative works of the Software, and to permit
+# third-parties to whom the Software is furnished to do so, all
+# subject to the following:
+
+# The copyright notices in the Software and this entire statement,
+# including the above license grant, this restriction and the following
+# disclaimer, must be included in all copies of the Software, in whole
+# or in part, and all derivative works of the Software, unless such copies
+# or derivative works are solely in the form of machine-executable object
+# code generated by a source language processor.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+# OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE, TITLE AND NON-INFRINGEMENT. IN NO EVENT SHALL
+# THE COPYRIGHT HOLDERS OR ANYONE DISTRIBUTING THE SOFTWARE BE LIABLE FOR ANY
+# DAMAGES OR OTHER LIABILITY, WHETHER IN CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+# IN THE SOFTWARE.
```

Comment:
I think it is sufficient to mention the license file as we did for other source code files:
```py
# Distributed under the Boost Software License, Version 1.0. (See
# accompanying file LICENSE_1_0.txt or copy at
# http://www.boost.org/LICENSE_1_0.txt)
```

Reply:
```suggestion
```